cli/src/config/templates.toml

```diff
@@ -31,12 +31,7 @@ separate(" ",
 ) ++ ": " ++ content
 '''
 
-config_list = '''
-if(overridden,
-  label("overridden", indent("# ", name ++ " = " ++ value)),
-  name ++ " = " ++ value,
-) ++ "\n"
-'''
+config.list = 'builtin_config_list'
```

Comment:
was this changed from `config_list` to `config.list` on purpose?

Reply:
Yes, should I revert?

Reply:
I suppose it's better to keep the key the same in case folks have overridden it---reverted

Reply:
We use underscores for `bookmark_list` and `file_annotate` above, so I think we should use that here too for consistency.